spec/dummy/app/models/concerns/password_concern.rb

```diff
@@ -0,0 +1,53 @@
+require "manageiq-password"
+
+module PasswordConcern
```

Comment:
Don't duplicate PasswordConcern here.  It is already of the ManageIQ::Password gem and that module can just be used directly.